documents/Libraries/stdlib/osl/mx_multiply_float_matrix33.osl

```diff
@@ -0,0 +1,18 @@
+// Open Shading Language : Copyright (c) 2009-2017 Sony Pictures Imageworks Inc., et al.
+// https://github.com/imageworks/OpenShadingLanguage/blob/master/LICENSE
+//
+// MaterialX specification (c) 2017 Lucasfilm Ltd.
+// http://www.materialx.org/
+
+#include "mx_funcs.h"
+
+shader mx_multiply_float_matrix33
+    [[ string help = "Matrix33"" multiply in1 by in2" ]]
+  (
+    matrix in1 = matrix(0),
```

Comment:
Another case of matrix3x3. I think in general matrix was not replaced by matrix33 ?

Reply:
Yes, another one.